### PR TITLE
Feature/check country eu membership

### PR DESCRIPTION
**Pull request by julianpollmann**
Fix for Enhancement #113

### CODE REVIEW

File: utils/ootest.php [hunk 1]
```diff
@@ -73,6 +73,11 @@
  if($myCountry->IsSEPA()) { print "Yes"; } else { print "No"; }
  print ".\n";
 
+ # output remaining country properties
+ print "Is a SEPA member? ";
+ if($myCountry->IsEuMember()) { print "Yes"; } else { print "No"; }
```
Comment (globalcitizen) [inline utils/ootest.php:78]: Looks like an issue here.

**Comment by globalcitizen** [comment]
Wow that was a quick turnaround! I am exhausted and need to check this while mentally awake tomorrow before pushing on users. Looks pretty good so far. A few things:
- Found one minor issue (see comment)
- Please define the field meanings somewhere.
- Please add documentation to `README.md`
- Please double check all scripts have been updated with the new field.

**Comment by julianpollmann** [comment]
Tests have been fixed and Readme is updated ;)

**Comment by globalcitizen** [comment]
Cheers, looks good. Merging.

As a new contributor if you're motivated to check for new IBAN registry entries, we're probably overdue. Just checck the PDF and see recent changes / new countries. Mostly it's only new countries, rarely do they make substantive changes to established ones. They used to publish a changelog, I think they stopped at some point when they internally changed maintainers. I would expect one or two by now, potentially.

**Pull request merged**